config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -299,18 +299,20 @@ fields:
 ## GERMPLASM
 ###
 
-    -   name: exp_germgenus
+    -   name: exp_organism
         content_type: research_experiment
-        label: Germplasm Genus
-        type: chado_property_type_default
-        description: The genus of the germplasm that data was collected for (i.e. that the experiment was done on). If data was collected for multiple genuses then each genus should be its own entry.
+        label: Germplasm Species
+        type: project_genus
+        description: The species of the germplasm that data was collected for (i.e. that the experiment was done on). If data was collected for multiple species then each species should be its own entry.
         cardinality: -1
         required: false
         storage_settings:
             storage_plugin_id: chado_storage
             storage_plugin_settings:
                 base_table: project
                 prop_table: projectprop
+            genus_term: genus (TAXRANK:0000005)
+            sciname_term: scientific name (NCBITaxon:scientific_name)
```

Comment:
This is now in the wrong place 🙈 It is now a field setting now a storage setting.